Project.toml

```
@@ -14,5 +14,5 @@ SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 [compat]
 AbstractMCMC = "2, 3"
 DensityInterface = "0.4"
-Setfield = "0.7.1, 0.8"
-julia = "1"
+Setfield = "0.8.1"
+julia = "1.6, 1.7"
```

This is equivalent to
```suggestion
julia = "1.6"
```